version.tf

```diff
@@ -6,5 +6,9 @@ terraform {
       source  = "ibm-cloud/ibm"
       version = ">= 1.70.0, <2.0.0"
     }
+    restapi = {
+      source  = "Mastercard/restapi"
+      version = "1.20.0"
```

Comment:
Module should use `>=` syntax and add an upper limit to prevent major version updates